packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -45,20 +45,18 @@ const BASE_CHAIN_ID = 'eip155'
 const MAX_PROVIDERS_COUNT = 100
 const TRANSACTION_CACHE_SIZE = 50
 const BLOCK_CACHE_SIZE = 50
-const V0_PROOF_TYPE = 'raw'
 const V1_PROOF_TYPE = 'f(bytes32)' // See: https://namespaces.chainagnostic.org/eip155/caip168
 
 const ABI = ['function anchorDagCbor(bytes32)']
 
 const iface = new Interface(ABI)
 
-//TODO (NET-1659): Finalize block numbers and smart contract addresses once CAS is creating smart contract anchors
 const BLOCK_THRESHHOLDS = {
```

Comment:
Where is the threshold for the dev-unstable network? I believe it's on sepolia?

Reply:
dev-unstable is on goerli

Reply:
oh that's right!